Mergin/project_status_dialog.py

```diff
@@ -204,7 +205,15 @@ def show_changes(self):
         if not self.changes_summary:
             self.ui.messageBar.pushMessage("Mergin", "No changes found in the project layers.", Qgis.Info)
             return
-
+        proceed, reply_btn = unsaved_project_check()
+        if not proceed:
+            return
+        if reply_btn and reply_btn != QMessageBox.Yes:
+            iface.messageBar().pushMessage(
```

Comment:
Same as above. Better to show this message in the project satus dialog. In current implementation messagebar partialy hidden and not very readable

Reply:
Done, but message is showing in the diff viewer dialog (same as for comment above)